include/rapid_pbd/motion_planning.h

```diff
@@ -35,6 +35,7 @@ class MotionPlanning {
   void ClearGoals();
   void BuildGoal(moveit_msgs::MoveGroupGoal* goal) const;
   int num_goals() const;
+  void PublishCollisionObject(moveit_msgs::CollisionObject&);
```

Comment:
The argument should be `const moveit_msgs::CollisionObject& obj`. In general, parameters that are not modified and used purely as inputs should be passed in as a const reference, while objects that are modified are passed as pointers.